drivers/i2c/i2c_ite_it8xxx2.c

```diff
@@ -730,14 +753,8 @@ static int i2c_it8xxx2_transfer(const struct device *dev, struct i2c_msg *msgs,
 		data->ridx = 0;
 		data->err = 0;
 		data->msgs = &(msgs[i]);
-		data->addr_16bit = addr;
+		data->addr_16bit = addr << 1;
```

Comment:
The TRASLA register already splits the I2C target address into ADR [7:1] and R/W [0] fields.
Perform this shift when writing to the TRASLA register to better match the datasheet.

IT83XX_SMB_TRASLA(base) = (uint8_t)(data->addr_16bit << 1 ) | 0x01;

Do the same thing when calling i2c_pio_trans_data.

Reply:
That appears to be a bad link.

Reply:
What about this link? https://github.com/zephyrproject-rtos/zephyr/pull/34809/commits/8f70f58c90eeb9ea303f07a6f30adfd862803337